data/cmdline/find-orfs.uwl

```diff
@@ -72,6 +72,10 @@ workflow find-orfs{
                 alias:name;
                 description:"Name of the result annotations marking found ORFs (using 'ORF' by default)";
             }
+            orf-search.strand {
+                alias:strand;
+                description:"Strand to search in";
```

Comment:
Please add 
1) Name of the options, so user knows which options should be passed in the command line.
2) Default option info